apps/nextjs/src/components/AppComponents/Chat/chat-list.tsx

```diff
@@ -81,6 +86,35 @@ export function ChatList({
 
   return (
     <div className="relative flex w-full flex-col " onScroll={handleScroll}>
+      <ChatMessagesDisplay
+        messages={messages}
+        id={id}
+        lastModeration={lastModeration}
+        persistedModerations={persistedModerations}
+        ailaStreamingStatus={ailaStreamingStatus}
+      />
+
+      {isDemoLocked && <DemoLimitMessage id={id} />}
```

Comment:
@mikeritson-oak We should test that the demo locking functionality still works as expected